gems/aws-sdk-core/lib/aws-sdk-core/rest/request/content_type.rb

```diff
@@ -1,51 +0,0 @@
-# frozen_string_literal: true
-
-module Aws
-  module Rest
-    # NOTE: headers could be already populated if specified on input shape
-    class ContentTypeHandler < Seahorse::Client::Handler
```

Comment:
I don't think we can remove this.  This will break old service w/ new core

Reply:
Edit: Actually nvm, this isn't dependend on by service clients directly, but through the protocol plugin.  So we may be okay.

Reply:
It hasn't released yet. This is new code too